js/interface.js

```diff
@@ -804,11 +805,17 @@
   attachObservers();
   // Load menu widgets on startup
   loadCustomMenuWidgets();
+
+  // to open menu links
+  if (JSON.parse(Fliplet.Navigate.query.data).links === 'true') {
```

Comment:
You should be able to use `if (Fliplet.Widget.getData().links)`. The data returned from `Fliplet.Widget.getData()` contains data passed through the `data` query parameter. See https://developers.fliplet.com/API/core/widget.html#get-a-widget-instance-data

Reply:
used Fliplet.Widget.getData().links in if statement to assess payload data